bin/create_release.sh

```diff
@@ -64,5 +68,6 @@ curl \
     --data-binary "@cumulus-orca-terraform.zip" \
     -H "Content-type: application/octet-stream" \
     $RELEASE_URL/assets?name=cumulus-orca-terraform.zip
```

Comment:
we should also validate that the $RELEASE_URL is valid. 
I have seen error in the past where the it shows error due to invalid url